src/Projects/ProjectService.cs

```diff
@@ -17,14 +17,12 @@ public class ProjectService : IProjectService
         public static readonly string ITEM_ENDPOINT = $"http://{Backend.BASE_URL}/api/items";
 
         private IAuthentication _auth;
-        private IStorageProvider _storage;
         private IMessageProvider _message;
         private IPhysicalFactory _physical;
 
-        public ProjectService(IAuthentication auth, IStorageProvider storage, IMessageProvider message, IPhysicalFactory physical)
```

Comment:
Assuming temporary, hence not deleting the interface?

Reply:
Sorry, dont folow?

Reply:
The library still has the interface IStorageProvider. I'm assuming you've leaving this as a place holder for future use.

Reply:
Ah thought you meant the class. Yes, left in for now as might be used to persist caching when added.